appeals/web/src/client/components/file-uploader/__tests__/file-uploader.test.js

```diff
@@ -7,7 +7,6 @@
 /* eslint-disable no-undef */
 import { fireEvent, waitFor } from '@testing-library/dom';
 import '@testing-library/jest-dom';
-import initFileUploaderModule from '../file-uploader.module.js';
```

Comment:
We need a ticket to investigate the issue and reinstate this test.

It can be linked to the Applications BO spike on the same issue: https://pins-ds.atlassian.net/browse/APPLICS-492

Reply:
https://pins-ds.atlassian.net/browse/BOAT-1420